http/server/internal.go

```diff
@@ -11,19 +11,19 @@ import (
 	"github.com/microdevs/missy/log"
 )
 
-type metric struct {
+type internal struct {
 	server
 
 	health    *health.Handler
 	readiness *readiness.Handler
 }
 
-func newMetric(c Config, l log.FieldsLogger) *metric {
-	s := &metric{
-		// TODO what about TLS on metric external?
+func newInternal(c Config, l log.FieldsLogger) *internal {
+	s := &internal{
+		// TODO what about TLS on internal external?
```

Comment:
Do we want to leave `TODO`s in code or maybe create an issue for that?

Reply:
it a prototype branch `v1` so I am not sure if before it will be in master it is a good idea to create issue... Let's leave this and address asap... For now we probably does not need TLS for internal server... and even if... it's cert/key can differ... 
We can leave this or add possibility to select TLS cert/key for internal as well and fix this now... 
I am not sure if this is important for now...